config/samples/resources/alloydbcluster/regular-cluster/alloydb_v1beta1_alloydbcluster.yaml

```diff
@@ -55,4 +55,4 @@ spec:
   initialUser:
     user: "postgres"
     password:
-      value: "postgres"
```

Comment:
Thank you for updating this. 
This field seems to contain sensitive data. I think a better approach is to use k8s secret to [store this data](https://cloud.google.com/config-connector/docs/how-to/secrets). 
Not a blocker of this PR, this is something need to be considered to switch.